presentation/src/main/java/com/example/healthysalad/SelectIngredientFragment.kt

```diff
@@ -8,67 +8,132 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
+import android.widget.TextView
+import androidx.core.os.bundleOf
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.example.data.Data
-
+import kotlinx.android.synthetic.main.fragment_select_ingredient.view.*
 
 /**
  * A simple [Fragment] subclass.
  * Use the [SelectIngredientFragment.newInstance] factory method to
  * create an instance of this fragment.
  */
-class SelectIngredientFragment (ingredientType : String) : Fragment() {
-    private val ingredientType = ingredientType
+
+private const val INGREDIENTTYPE  = "base"
+
+class SelectIngredientFragment () : Fragment() {
+
+    private var type: String = "base"
```

Comment:
Same here. We could change this line to be
`private var type: String = INGREDIENTTYPE` or 
`private var type: String = INGREDIENTTYPE_BASE`

Reply:
yup its just the initial value. it will change later